src/postgresql_training/insert_rows_cont.py

```diff
@@ -0,0 +1,62 @@
+from meta import MyDB
+
+import pandas as pd
+import psycopg2
+from sklearn.datasets import load_iris
+import time
+
+# Load iris dataset
+iris_dataset = load_iris()
+features, labels = iris_dataset["data"], iris_dataset["target"]
+feature_names = ["sepal_length_cm", "sepal_width_cm", "petal_length_cm", "petal_width_cm"]
+label_names = ["target"]
+
+feature_df = pd.DataFrame(features, columns=feature_names)
+label_df = pd.DataFrame(labels, columns=label_names)
+
+df = pd.concat([feature_df, label_df], axis=1)
+
+
+def data_generator(df: pd.DataFrame):
+    """
+    Data generator
+    """
+    idx = 0
+    while idx < len(df):
+        yield tuple(df.iloc[idx].values)
+        idx += 1
```

Comment:
```suggestion
    while True:
        yield tuple(df.iloc[idx].values)
        idx += 1
        idx %= len(df)
```

더한 후에 바로 나머지 처리를 해주면 idx 를 초기화할 필요는 없지 않을까요?

Reply:
반영하였습니다 :)